ci/docker/Dockerfile.build2.ubuntu_gpu_cu111

```diff
@@ -0,0 +1,44 @@
+# -*- mode: dockerfile -*-
```

Comment:
Is this the correct name for the dockerfile? Why does it contain "build2" instead of just "build"?

Reply:
copy paste problems with eclipse :P